TaskLayer/SearchTask/SearchTask.cs

```diff
@@ -711,7 +711,8 @@ protected override MyTaskResults RunSpecific(string OutputFolder, List<DbForTask
                 ionTypes.Add(ProductType.Zdot);
             if (CommonParameters.CIons)
                 ionTypes.Add(ProductType.C);
-            TerminusType terminusType = ProductTypeToTerminusType.IdentifyTerminusType(ionTypes);
+            TerminusType terminusTypeForNonSpecificSearch = (SearchParameters.SearchType == SearchType.NonSpecific) ? ProductTypeToTerminusType.IdentifyTerminusType(ionTypes) : TerminusType.None;
```

Comment:
Not a fan of keeping this field around in the search task in general. Could it be collapsed in the if statement on line 745?